docs/using/visualization/generating_widgets.ipynb

```diff
@@ -574,6 +574,19 @@
     "![Line Info Widget Demo](../images/line_info_widget_demo.gif)"
```

Comment:
<p>Looks good - just remove the heading "Enabling ...", there's no need of it since we're already using an information box. Another correction: "... must be active in order to show virtual packets' spectrum ..." - "to produce to show" is repetitive.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1575/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>